runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -270,13 +273,18 @@ public CorfuRuntime() {
         log.info("Corfu runtime version {} initialized.", getVersionString());
     }
 
+    public CorfuRuntime(String configurationSring) {
+        this(configurationSring, getDefaultMetrics());
+    }
+
     /**
      * Parse a configuration string and get a CorfuRuntime.
      *
      * @param configurationString The configuration string to parse.
      */
-    public CorfuRuntime(String configurationString) {
+    public CorfuRuntime(String configurationString, MetricRegistry metricRegistry) {
```

Comment:
@zalokhan I think that Maithem's suggestion is good.  It seems to me reasonable to assume that if you wish to use a custom registry, then it must be configured prior to calling connect().  The same goes for many of the other set'able runtime values and also configuring the use of TLS.